src/components/cards/PairPriceGraph/PairPriceGraph.vue

```diff
@@ -320,7 +320,7 @@ const chartGrid = computed(() => {
               :showLegend="false"
               :color="chartColors"
               :customGrid="chartGrid"
-              :axisLabelFormatter="{ yAxis: '0.000000' }"
+              :axisLabelFormatter="{ yAxis: { maximumFractionDigits: 6 } }"
```

Comment:
@alter-eggo  Wanted to make sure if this is this correct now?

Reply:
yAxis refers to fiat value here, this suggestion may be better here:
```suggestion
              :axisLabelFormatter="{ yAxis: { style: 'currency' } }"
```